github/apps_marketplace_test.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+func TestMarketplaceService_ListPlans(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/marketplace_listing/plans", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", mediaTypeMarketplacePreview)
+		testFormValues(t, r, values{
+			"page":     "1",
+			"per_page": "2",
+		})
+		fmt.Fprint(w, `[{"id":1}]`)
+	})
+
+	opt := &ListOptions{Page: 1, PerPage: 2}
+	client.Marketplace.Stubbed = false
```

Comment:
Hmmm... I wish we didn't use a package global `client` for unit testing, as this will prevent unit tests from all being run in parallel in the future.

Ideally, we would have something like:

```go
client, teardown := setup()
defer teardown()
```

for each test.  No action is needed in this PR, but I'll file a new issue and point here for motivation.